app/assets/javascripts/views/file_uploader_view.js.coffee

```diff
@@ -26,6 +27,9 @@ class ELMO.Views.MediaUploaderView extends ELMO.Views.ApplicationView
   events:
     'click .existing a.delete': 'delete_existing'
 
+  set_listener: (listener) ->
```

Comment:
Would prefer to just pass in the listener. Made this method because in media_editable.html, the params are passed in in ruby, and referencing the JS object in ruby isn't working. Not sure if this is just a syntax thing, or a js/ruby issue.